### ISSUE TITLE
Compatibily to Prestashop 1.7.7.8

### ISSUE THREAD

**Issue by totti240282** [Bukimedia/PrestaSharp#428]
is it possible to make the library compatible with the latest version?

**Comment by jacktric** [comment]
Are there many problems with the last Prestashop version? 
I'm new to Prestashop and I'm just looking for a .NET client.

**Comment by totti240282** [comment]
Yes.

**Comment by AssCasc** [comment]
Are you sure that the version number is 1.7.7.8, i think the problems start from the 1.7.8.0 version, however i've a lot of problem in creating/updating too, someone could fix?

**Comment by sasciamorelli** [comment]
has anyone figured out what the problem can be?

**Comment by jacktric** [comment]
I'm using the library with version 1.7.7.8 without problems.
I Add/Update products, images and available-stocks.

**Comment by sasciamorelli** [comment]
in fact, it is version 1.7.8.0 that is the last that does not work with prestasharp

**Comment by AssCasc** [comment]
True, the 1.7.7.8 version correctly work but the 1.7.8.0 version do not

**Comment by AssCasc** [comment]
According to this issue in official prestashop github https://github.com/PrestaShop/PrestaShop/issues/26361 if you are setting, as i was, the position_in_category field you must:
```
For solving it :
- You must set id_category_default with an ID AND position_in_category with a value > 1.
- OR you must unset id_category_default AND position_in_category.
```

**Comment by IcuScc** [comment]
> According to this issue in official prestashop github [PrestaShop/PrestaShop#26361](https://github.com/PrestaShop/PrestaShop/issues/26361) if you are setting, as i was, the position_in_category field you must:
> 
> ```
> For solving it :
> - You must set id_category_default with an ID AND position_in_category with a value > 1.
> - OR you must unset id_category_default AND position_in_category.
> ```

Hello. In my case i unset the property position_in_category... and i can create product...

**Comment by IcuScc** [comment]
With Prestashop 1.7.8.X and PHP 7.4... after product is created, i can not get using a filter by reference. It returns to me zero results.
if i create the product by hand in prestashop, the filter by reference works ok.

Any idea?

Thanks

**Comment by IcuScc** [comment]
Hi good morning to all

I changed the way of filter

    productosPs = PsProducto.GetByFilter(psFiltroRef, null, null);
    by
    productosPs = PsProducto.GetByFilter(psFiltroRef, null, "1");

using the limit parameters works for me the cerate and update products

i hope this answer works for you aswell.

see you
